src/lib/config/defaultConfig.js

```diff
@@ -10,7 +10,7 @@ import {
 export const APP_CONFIG = {
   ENABLE_LOCAL_ACCOUNT_LOGIN: true,
   ENABLE_OAUTH_LOGIN: true,
-  ES_DELAY: 3000,
+  SEARCH_READY_DELAY: 2000,
```

Comment:
will it have any impact for "slower" machines?

Reply:
it should not: by default ES has refresh time 1 sec, it should be independent of the machines...